pkg/ioutil/ioutil.go

```diff
@@ -114,3 +114,15 @@ func (w *LimitWriter) Close() error {
 func LimitedWriter(w io.Writer, skipBytes int64, limit int64) *LimitWriter {
 	return &LimitWriter{w, skipBytes, limit}
 }
+
+type nopCloser struct {
+	io.Writer
+}
+
+func (nopCloser) Close() error { return nil }
+
+// NopCloser returns a WriteCloser with a no-op Close method wrapping
+// the provided Writer w.
+func NopCloser(w io.Writer) io.WriteCloser {
```

Comment:
I guess NopWriteCloser is better than NopCloser because golang already has it to avoid confusion

Reply:
Standard lib is for `io.Reader` :smiley:

Reply:
ioutil itself is a conflict there is no governing body for these things, this NopCloser is for writer due to the signature i would like to keep it since original one doesn't say NopReadCloser()